drv/Function.drv

```diff
@@ -104,9 +159,13 @@ public interface FUNCTION KEY_VALUE_GENERIC extends Function<KEY_GENERIC_CLASS,
 
 	/** {@inheritDoc}
 	 * @deprecated Please use the corresponding type-specific method instead. */
+
 	@Deprecated
 	@Override
-	boolean containsKey(Object key);
+	default boolean containsKey(Object key) {
```

Comment:
This is different from the original methods, which just checks for null (returning null).